### PR TITLE
v1.1 release notes

### PR DESCRIPTION
**Pull request by denniszollo**
@cbeighley @switanis @jdormoy @swift-nav/business

### CODE REVIEW

File: release_notes.txt [hunk 1]
```diff
@@ -0,0 +1,104 @@
+===============================================================================
+                              Swift Navigation, Inc.
+                      Piksi Multi Firmware Release Notes
+
+Product Page:      swiftnav.com/piksi-multi
```
Comment (wltr) [inline release_notes.txt]: `http://swiftnav.com/piksi-multi`?

File: release_notes.txt [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+* [234] NMEA GSA message reports used SVs.
+* [274] NMEA GSV message reports visible SVs.
+* [283] SBP messages MSG_UTC_TIME, MSG_DGPS_STATUS, MSG_AGE_CORRECTIONS have been implemented.
+
+KNOWN ISSUES:
+* [252] 1 PPS output is still generated after fix is lost.
+* [287] Incoming RTK corrections are echoed on the same port. It can create additional data traffic over the radio.
+* [230] Ethernet port LED does not show activity during communication.
+* [227] LED indicators may show incorrect status during reset.
+* [282] External event input has no debouncing. A sharp input edges are required to avoid multiple messages with
```
Comment (wltr) [inline release_notes.txt]: Hardware has de-bouncing filtering glitches shorter than 500 ns. 
Software de-bouncing on microsecond basis was added with https://github.com/swift-nav/piksi_firmware_private/pull/594

File: release_notes.txt [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+Product Page:      swiftnav.com/piksi-multi
+Firmware Download: support.swiftnav.com
+===============================================================================
+
+
+-------------------------------------------------------------------------------
+Release 1.1.26                                                       2017-05-16
+-------------------------------------------------------------------------------
+FEATURES:
```
Comment (benjamin0) [inline release_notes.txt]: Should we enumerate new features at the top? Heading, ntrip, rtcm in, usb logging, etc.

File: release_notes.txt (hunk 2 above)
Comment (wltr) [inline release_notes.txt]: There seems to be a new issue now though. 
https://github.com/swift-nav/piksi_v3_bug_tracking/issues/465

Comment (denniszollo) [inline release_notes.txt]: can you check now and see if what I wrote reflects your current understanding of the feature?

File: release_notes.txt (hunk 3 above)
Comment (denniszollo) [inline release_notes.txt]: I swapped Features for Updates and renamed updates to NEW FEATURES AND UPDATES:

**Pull request merged**